packages/pieces/community/file-helper/src/index.ts

```diff
@@ -11,6 +12,6 @@ export const filesHelper = createPiece({
   logoUrl: 'https://cdn.activepieces.com/pieces/file-piece.svg',
   categories: [PieceCategory.CORE],
   authors: ['kishanprmr', 'MoShizzle', 'abuaboud'],
```

Comment:
Add yourself here, so you are shown as a contributor on the piece on our website :D

Reply:
Nice, done: https://github.com/activepieces/activepieces/pull/6098/files#diff-cd4c9d71cedac0af6d7e9986944bf179863ada093101a3d5e77ab82f6982919bR14